pages/player/[id].tsx

```diff
@@ -0,0 +1,109 @@
+import { useState, useEffect } from "react";
+import { Player } from "@prisma/client";
+import DashboardLayout from "components/DashboardLayout";
+import PlayerNavBar from "pages/player/PlayerNavBar";
+import { useRouter } from "next/router";
+
+interface playerProp {
+  playerProp: Player & {
+    user: {
+      name: string | null;
+    };
+  };
+}
+
+const GetID = (): string | string[] | undefined => {
+  const router = useRouter();
+  const { id } = router.query;
+  return id;
+};
+
+const emptyPlayer: Player & {
+  user: {
+    name: string | null;
+  };
+} = {
+  user_id: 0,
+  bio: null,
+  academicEngagementScore: null,
+  academicEngagementComments: null,
+  advisingScore: null,
+  advisingComments: null,
+  athleticScore: null,
+  athleticComments: null,
+  advisingAbsences: null,
+  athleticAbsences: null,
+  gpa: null,
+  disciplinaryActions: null,
+  schoolAbsences: null,
+  bmi: null,
+  beepTest: null,
+  mileTime: null,
+  healthAndWellness: null,
+  highlights: null,
+  user: { name: null },
+};
+
+const PlayerProfile: React.FunctionComponent = () => {
+  const [player, setPlayer] = useState<
+    Player & {
+      user: {
+        name: string | null;
+      };
+    }
+  >(emptyPlayer);
+  const ID = GetID();
+  useEffect(() => {
+    async function getPlayer(): Promise<void> {
+      try {
+        const response = await fetch(
+          `http://localhost:3000/api/players/read/${ID}`,
+          {
+            method: "GET",
+            headers: { "content-type": "application/json" },
+          }
+        );
+        const player1: Player & {
```

Comment:
let's rename this to playerdata for clarity

Reply:
made all changes!